app/rtc_s1_parallel.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python
+
+'''
+RTC Workflow
+'''
+
+import argparse
+import multiprocessing
+import os
+import subprocess
+import time
+import yaml
+
+from rtc.runconfig import RunConfig
+
+
+def get_rtc_s1_parser():
+    '''Initialize YamlArgparse class and parse CLI arguments for OPERA RTC.
+    Modified after copied from `rtc_s1.py`
+    '''
+    parser = argparse.ArgumentParser(description='',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('run_config_path',
+                        type=str,
+                        nargs='?',
+                        default=None,
+                        help='Path to run config file')
+
+    parser.add_argument('-n',
+                        dest='num_workers',
+                        type=int,
+                        default=4, # TODO: Discuss if the default # worker makes sense.
```

Comment:
Can you have the default value for `num_workers` be the number of available threads? Not sure if that makes sense?

Reply:
I'll make the default for `num_workers` as `os.cpu_count()`. What do you think about running some tests later to find the optimal number of workers?

Reply:
Uhm, no, we need to use the number of available threads. I usually set up (limit) the number of threads to use by setting the `OMP_NUM_THREADS` system variable. For example:
```
export OMP_NUM_THREADS=32
```
That means that we'll have at most 32 threads available for processing. We need somehow to read the number of available threads.

Reply:
Updated the procedure to determine the default `num_workers` based on the discussion in the Slack chat.